app/models/katello/pool_with_quantities.rb

```diff
@@ -0,0 +1,11 @@
+module Katello
+  class PoolWithQuantities
```

Comment:
I know this is a rather simple class, but given it's used in a number of spots I think it may be worth a few unit tests.